lib/hayfork/binding.rb

```diff
@@ -58,6 +58,18 @@ def quoted_value
       end
     end
 
+  private
+
+    def types_method
+      before_rails61? ? :types : :klass
+    end
+
+    def before_rails61?
+      return true if ActiveRecord::VERSION::MAJOR < 6
+
+      ActiveRecord::VERSION::MINOR < 1
```

Comment:
```suggestion
      ActiveRecord.version < Gem::Version.new("6.1") ? :types : :klass
```

like in `hayfork/statement.rb`?

Reply:
Updated the code in this spot as well. 👍